### DISCUSSION TITLE
My header & footer builder doesn't work

### DISCUSSION THREAD

**Discussion by jbtmetals** [community/community#46826] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

It happens suddenly, when I editing a page on my wordpress website, it failed to save the page and says 503 erro.
I can not access my homepage while other pages seems fine. I can visit wordpress dashboard but can not edit anything with elementor.
I run a debug, the results are as follows:

[09-Feb-2023 01:06:53 UTC] PHP Deprecated:  Function Elementor\Widgets_Manager::register_widget_type is <strong>deprecated</strong> since version 3.5.0! Use register instead. in /home/u110017121/domains/jbtmetals.com/public_html/wp-includes/functions.php on line 5383

when I deactive header & footer builder, I can visit my website. I tried to roll back elementor version, reinstalled elementor and the head footer builder, still can not get it work. please help.

I copied the code around line 5383

	 * @param bool $trigger Whether to trigger the error for deprecated functions. Default true.
	 */
	if ( WP_DEBUG && apply_filters( 'deprecated_function_trigger_error', true ) ) {
		if ( function_exists( '__' ) ) {
			if ( $replacement ) {
				trigger_error(
					sprintf(
						/* translators: 1: PHP function name, 2: Version number, 3: Alternative function name. */
						__( 'Function %1$s is <strong>deprecated</strong> since version %2$s! Use %3$s instead.' ),
						$function,
						$version,
						$replacement
					),
					E_USER_DEPRECATED
				);
			} else {
				trigger_error(
					sprintf(
						/* translators: 1: PHP function name, 2: Version number. */
						__( 'Function %1$s is <strong>deprecated</strong> since version %2$s with no alternative available.' ),
						$function,
						$version
					),
					E_USER_DEPRECATED
				);
			}
		} else {
			if ( $replacement ) {
				trigger_error(
					sprintf(
						'Function %1$s is <strong>deprecated</strong> since version %2$s! Use %3$s instead.',
						$function,
						$version,
						$replacement
					),
					E_USER_DEPRECATED
				);
			} else {
				trigger_error(
					sprintf(
						'Function %1$s is <strong>deprecated</strong> since version %2$s with no alternative available.',
						$function,
						$version
					),
					E_USER_DEPRECATED

**Comment by garousiamir** [comment]
Hi,
I suggest if you have not yet created your header use this plugin:
https://wordpress.org/plugins/boostify-header-footer-builder/
it's amazing and better than elemetor default header footer builder
